test/routeBuilderSpec.js

```diff
@@ -91,6 +91,13 @@ describe('RouteBuilder', function () {
       secret: Buffer.from('VafuntVJRw6YzDTs4IgIU1IPJACywtgUUQJHh1u018w=', 'base64')
     })
     yield this.ledgers.connect()
+
+    // Populate the curve cache.
```

Comment:
Isn't this a local route? So it should already have a curve when it's added to the routing table?